web/src/main/java/org/openmrs/web/filter/GZIPFilter.java

```diff
@@ -124,9 +125,9 @@ private boolean isGZIPEnabled() {
 		}
 		
 		try {
+			Context.addProxyPrivilege(PrivilegeConstants.GET_GLOBAL_PROPERTIES);
```

Comment:
We could also just catch the APIAuthenticationException here and log it, before returning false, in order to reduce the need to add proxy privileges. This is based on the fact that it is not a big deal for a not yet logged in user to miss gzip compression. They can have it after login.

Reply:
@wikumChamith i am changing my mind on this. I think it would be better to add the proxy privilege to avoid these unnecessary errors before login. :)